src/main/java/io/airbyte/testingtool/scenario/action/connection/ActionCreateConnection.java

```diff
@@ -65,4 +119,24 @@ protected ConnectionCreate getConnectionCreateConfig() throws ApiException {
         .namespaceFormat("output_namespace_${SOURCE_NAMESPACE}")
         .prefix("output_table_");
   }
+
+  protected void updateUseNormalizationFlag(boolean value) {
```

Comment:
Such update methods have high requirements during code changing. You need not forget to call them and know when you should do that.
So, please replace it by method like : `protected void getNormalizationFlag()`
It will return always `true` in this class and `Objects.nonNull(getSettings().getNormalization()) ? getSettings().getNormalization() : true` in the custom action.

Reply:
done, thanks for the comment